src/views/UrbitID/ResetKeys/ResetExecute.js

```diff
@@ -92,20 +96,33 @@ export default function ResetExecute({ newWallet, setNewWallet }) {
         walletType === WALLET_TYPES.WALLET_CONNECT ? wcSend : undefined;
 
       try {
-        await reticketPointBetweenWallets({
-          fromWallet: need.wallet(wallet),
-          fromWalletType: walletType,
-          fromWalletHdPath: walletHdPath,
-          toWallet: newWallet.value.wallet,
-          point: point,
-          web3: need.web3(web3),
-          contracts: need.contracts(contracts),
-          networkType,
-          onUpdate: handleUpdate,
-          nextRevision: networkRevision + 1,
-          txnSigner,
-          txnSender,
-        });
+        if (currentL2) {
+          console.log(newWallet.value);
+          // FIXME: the useEffect is called twice, which is fine since the duplicate
```

Comment:
Since this is not a showstopper, we can ignore it for now, and I can take a look at this in a follow up PR